game/scripts/npc/abilities/antimage_mana_void.txt

```diff
@@ -47,7 +47,7 @@
       "01"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "mana_void_damage_per_mana"                       "0.8 1.1 1.4 1.85 2.3" //OAA
+        "mana_void_damage_per_mana"                       "0.8 0.95 1.1 1.25 1.4"
```

Comment:
im pretty sure old values were lik 1.3 1.6 but i cant remember

Reply:
> > Looked it up
> > 0.8 0.95 1.1 1.55 2.3 was old values.
> 
> I meant dota values lmao. No extra damage scaling at OAA levels. The damage scales when the enemy buys items. I think it should be good. I will test it before wednesday. My other idea for scaling was "0.8 1.0 1.2 1.4 1.6" if the damage felt bad.

My reason is was buffed cuz I complained the first two levels were bad and wanted those numbers buffed (I also didn't realized AOE was gimped 🤷 )

My values personally would be 1.0/1.2/1.4/1.75/2.0